googleapis/google-cloud-node
# migrate code from googleapis/nodejs-bigquery-reservation

- chore: add initial files for launch
- feat: generate v1 of the API (#2)
- chore: release 1.0.0 (#3)
- chore(deps): update dependency mocha to v8 (#4)
- chore: add dev dependencies for cloud-rad ref docs (#8)
- chore(deps): update dependency ts-loader to v8 (#5)
- build: use the latest build and docs config (#13)
- chore: start tracking obsolete files (#14)
- build(test): recursively find test files; fail on unsupported dependency versions (#15)
- chore: update bucket for cloud-rad (#17)
- build(node_library): migrate to Trampoline V2 (#18)
- chore(deps): update dependency webpack-cli to v4 (#21)
- chore(deps): update dependency webpack to v5 (#20)
- build: only check --engine-strict for production deps (#22)
- chore: clean up Node.js TOC for cloud-rad (#23)
- docs: updated code of conduct (includes update to actions) (#26)
- build(node): add KOKORO_BUILD_ARTIFACTS_SUBDIR to env (#27)
- fix: do not modify options object, use defaultScopes (#28)
- chore: release 1.0.1 (#29)
- fix(browser): check for fetch on window
- build: stop tracking generated files
- docs: spelling correction for "targetting" (#36)
- chore: release 1.0.2 (#34)
- fix(deps): roll back dependency @google-cloud/bigquery-reservation to ^1.0.1 (#38)
- chore: generate GAPIC metadata JSON file (#39)
- docs: add instructions for authenticating for system tests (#41)
- chore: update license headers (#42)
- feat: introduce style enumeration (#43)
- chore: release 1.1.0 (#44)
- refactor(nodejs): move build cop to flakybot (#46)
- chore: update CODEOWNERS config (#47)
- build: adds UNORDERED_LIST enum (#48)
- build: update gapic-generator-typescript to v1.2.10. (#49)
- chore(deps): update dependency sinon to v10 (#50)
- chore: migrate to owl bot (#51)
- chore(deps): update dependency @types/sinon to v10 (#55)
- fix(deps): require google-gax v2.12.0 (#64)
- chore: new owl bot post processor docker image (#66)
- fix: use require() to load JSON protos (#67)
- chore: update gapic-generator-typescript to v1.3.2 (#68)
- chore: release 1.1.1 (#65)
- chore(deps): update dependency sinon to v11 (#71)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#72)
- chore: release 1.1.2 (#73)
- chore(nodejs): remove api-extractor dependencies (#78)
- fix: make request optional in all cases (#80)
- chore: release 1.1.3 (#81)
- fix(deps): google-gax v2.17.0 with mTLS (#84)
- chore: release 1.1.4 (#86)
- fix(deps): google-gax v2.17.1 (#87)
- chore: release 1.1.5 (#88)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#89)
- chore: release 1.1.6 (#90)
- chore(nodejs): update client ref docs link in metadata (#95)
- fix(build): migrate to using main branch (#97)
- fix(deps): google-gax v2.24.1 (#99)
- feat: turns on self-signed JWT feature flag (#100)
- chore: release 1.2.0 (#101)
- feat: Deprecated SearchAssignments in favor of SearchAllAssignments
- chore: release 1.3.0 (#104)
- docs(samples): add auto-generated samples for Node with api short name in region tag (#109)
- build(node): update deps used during postprocessing (#1243) (#112)
- chore(cloud-rad): delete api-extractor config (#113)
- chore: fix the wrong post processor image (#115)
- chore(deps): update dependency @types/node to v16 (#114)
- chore(deps): update dependency sinon to v12 (#116)
- docs(samples): add example tags to generated samples (#117)
- build: add generated samples to .eslintignore (#118)
- docs(node): support "stable"/"preview" release level (#1312) (#122)
- feat: increase the logical timeout (retry deadline) to 5 minutes (#120)
- chore: release 1.4.0 (#123)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#124)
- chore: add api_shortname and library_type to repo metadata (#121)
- test(nodejs): remove 15 add 16 (#1322) (#126)
- chore: update v2.12.0 gapic-generator-typescript (#132)
- chore(deps): update dependency sinon to v13 (#134)
- docs(nodejs): version support policy edits (#1346) (#136)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#137)
- chore(deps): update dependency gts to v3 (#127)
- chore: update v2.14.2 gapic-generator-typescript (#141)
- docs: cleanup and clarifications (#147)
- build(node): update client library version in samples metadata (#1356) (#150)
- chore(deps): update dependency @types/mocha to v9 (#151)
- chore(deps): update dependency sinon to v14 (#154)
- build!: update library to use Node 12 (#155)
- chore(main): release 2.0.0 (#157)
- chore(deps): update dependency jsdoc-region-tag to v2 (#160)
- chore(deps): update dependency jsdoc-fresh to v2 (#159)
- fix(docs): document fallback rest option (#161)
- chore(deps): update dependency linkinator to v4 (#164)
- fix: better support for fallback mode (#165)
- fix: change import long to require (#166)
- fix: do not import the whole google-gax from proto JS (#1553) (#168)
- fix: allow passing gax instance to client constructor (#169)
- chore(main): release 2.0.1 (#162)
- fix: preserve default values in x-goog-request-params header (#170)
- build: use fully qualified request type name in tests (#173)
- chore(main): release 2.0.2 (#171)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
